src/sass/config/animation/_index.scss

```diff
@@ -0,0 +1,39 @@
+@use 'sassdoc-theme-herman/scss/utilities' as herman;
+@use '../tools';
+@use 'easing';
+@use 'times';
+@use 'sass:meta';
+@forward 'easing';
+@forward 'times';
+
+/// # Animation Config
+/// Accoutrement maps for storing global animation tokens.
+/// @link https://www.oddbird.net/accoutrement/docs/animate.html
+///   Accoutrement Animate
+/// @group animation
+
+/// ## Easing
+/// ---------
+/// Named easings that can be re-used to create consistent movement.
+/// @group animation
+/// @example scss
+///   @use 'config/animation/easing';
+///   @use 'config/tools';
+///   @use 'sass:meta';
+///
+///   @each $name, $easing in tools.compile-easing(meta.module-variables('easing')) {
+///   /* #{$name}: #{$easing}; */
+///   }
+
+/// ## Times
+/// ---------
+/// Named times that can be re-used to create consistent motion timing.
+/// @group animation
+/// @example scss
+///   @use 'config/animation/times';
+///   @use 'config/tools';
+///   @use 'sass:meta';
+///
+///   @each $name, $time in tools.compile-times(meta.module-variables('times')) {
+///   /* #{$name}: #{$time}; */
+///   }
```

Comment:
@dvdherron @mirisuzanne I might be missing something obvious, but I am getting an error on this and it's almost exactly what we have in Oddleventy (other than we are only using easing here instead of changes).

```
change:src/sass/config/animation/_easing.scss
9:09:24 PM [vite] page reload src/sass/config/animation/_easing.scss
» [WARNING] Error compiling @example scss: 
Can't find stylesheet to import.
  ╷
1 │ ┌ @forward 'accoutrement/sass/tools' with (
2 │ │   $color-var-prefix: '',
3 │ │   $easing-var-prefix: '',
4 │ │   $ratio-var-prefix: '',
5 │ │   $size-var-prefix: '',
6 │ │   $time-var-prefix: ''
7 │ │ );
  │ └─^
  ╵
  src/sass/config/_tools.scss 1:1  @forward
  _index.scss 1:1                  @use
  - 1:1                            root stylesheet
@use 'config';
@use 'config/animation/easing';
@use 'config/tools';
@use 'sass:meta';

@each $name, $easing in tools.compile-easing(meta.module-variables('easing')) {
/* #{$name}: #{$easing}; */
}
» [WARNING] Error compiling @example scss: 
Can't find stylesheet to import.
  ╷
1 │ ┌ @forward 'accoutrement/sass/tools' with (
2 │ │   $color-var-prefix: '',
3 │ │   $easing-var-prefix: '',
4 │ │   $ratio-var-prefix: '',
5 │ │   $size-var-prefix: '',
6 │ │   $time-var-prefix: ''
7 │ │ );
  │ └─^
  ╵
  src/sass/config/_tools.scss 1:1  @forward
  _index.scss 1:1                  @use
  - 1:1                            root stylesheet
@use 'config';
@use 'config/animation/times';
@use 'config/tools';
@use 'sass:meta';

@each $name, $time in tools.compile-times(meta.module-variables('times')) {
/* #{$name}: #{$time}; */
}
```

Reply:
@stacyk It looks like OddSite has a `nodeImporter` function that finds installed modules like Accoutrement and Herman with a shortcut (it looks for the `~` at the beginning of imported paths. It doesn't seem like that's set up in this project yet. 

I was able to get this to work by manually changing all of the Accoutrment and Herman imports to their respective directories in `node_modules`. So in `config/tools` the path would be `../../../node_modules/accoutrement/`. We probably could load `node_modules` in a similar way here too, right @jgerigmeyer?